guest-tools/td_virsh_tool.sh

```diff
@@ -212,6 +218,12 @@ clean_all() {
     exit 0
 }
 
+print_all() {
+    for domain_to_print in $(virsh list --all --name | grep ${DOMAIN_PREFIX}); do
```

Comment:
I think `virsh list --state-running --name` is better here to avoid including TDs that are shut off or in some other state that makes the guest inaccessible (the guests shouldn't be in this state but just in case). Otherwise you can end up with something like:

```
$ ./td_virsh_tool.sh --list
Domain td_guest-4fb86744-a222-4174-a428-43f64c2242b7 running with vsock CID: 3, ssh -p 36113 root@localhost
error: Requested operation is not valid: domain is not running
error: Requested operation is not valid: domain is not running
Domain td_guest-06087305-06c3-4b6c-958e-563f0fc29418 running with vsock CID: , ssh -p  root@localhost
error: Requested operation is not valid: domain is not running
error: Requested operation is not valid: domain is not running
Domain td_guest-5604771f-69fa-4fe6-aaa2-d5530674abba running with vsock CID: , ssh -p  root@localhost
```